content/akeneo-event-platform/platform-usage-requirements.md

```diff
@@ -6,7 +6,7 @@
 
 ## HTTPS destination
 
-The platform will test the HTTPS endpoint validitybefore being able to create the subscription.
+The platform will test the HTTPS endpoint validity before being able to create the subscription.
```

Comment:
To re-discuss, we said that this behavior should be removed.

Do not test the endpoint with HEAD request, not a problem, the first event we will try to deliver will suspend the subscription if 404. -> less logic in our code, less things to explain document, less things to develop/configure for integrators. cc @clovis-akeneo

Reply:
OK 
i removed it here also https://pr-910.api-dev.akeneo.com/akeneo-event-platform/concepts.html#configuration
does this page still relevant on its own as it carry few information ?

Reply:
You're right I'd rather move this "authentication/authorization" part as first paragraph of "Concepts" section

Reply:
due to the last addition from @val-- I kept the page but renamed it and moved it upper 
![image](https://github.com/user-attachments/assets/9ef21fff-0935-43d8-9d72-aa041a915902)